cities/hannover.json

```diff
@@ -56,7 +56,7 @@
             "properties": {
                 "title": "Wochenmarkt Döhren",
                 "location": "Fiedelerplatz, 30519 Hannover",
-                "opening_hours": "Fr 8:00-13:00"
+                "opening_hours": "Di 14:00-18:00"
```

Comment:
@katrinleinweber Did you see the error in the build log:

```
===> Validating cities/hannover.json ...
HANNOVER: Wochenmarkt Döhren
Error: Di <--- (Please use the English abbreviation "Tu" for "di".)
Validation done. 0 warning(s), 1 error(s) detected.
```

Reply:
Sorry for the delay! Done now in bb4484ceaad8651c25f99501b07a6597d63e7ab6 :-)